src/figuresClasses.ts

```diff
@@ -1,19 +1,81 @@
+
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
+
 export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number
+}
 
+function rounding(num: number): number {
+  return Math.floor(num * 100) / 100;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  public shape: Shape = 'triangle';
```

Comment:
so you can use it like this
```suggestion
  public shape: Shape = Shape.Triangle;
```